src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+import { HooksConfig } from "../../../playwright";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const tooltipText = "Some helpful text goes here";
+const colors = [
+  ["orange", COLOR.ORANGE],
+  ["red", COLOR.RED],
+  ["black", COLOR.BLACK],
+  ["brown", COLOR.BROWN],
+];
+
+test.describe("when focused", () => {
+  test("should have the expected styling when the focusRedesignOptOut is false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<HelpComponentTest />);
+
+    const elementLocator = await helpComponent(page);
+    await elementLocator.focus();
+
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+
+  test("should have the expected styling when the focusRedesignOptOut is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<HelpComponentTest />, {
+      hooksConfig: { focusRedesignOptOut: true },
+    });
+    const elementLocator = await helpComponent(page);
+    await elementLocator.focus();
+
+    await expect(helpComponent(page).locator("span")).toHaveCSS(
+      "outline",
+      "rgb(255, 188, 25) solid 2px"
+    );
+  });
+});
+
+test.describe("Testing Help component properties", () => {
+  testData.forEach((ariaLabel) => {
+    test(`should check ariaLabel as ${ariaLabel}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest ariaLabel={ariaLabel} />);
+      await expect(helpComponent(page)).toHaveAttribute(
+        "aria-label",
+        ariaLabel
+      );
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check className as ${className}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest className={className} />);
+
+      await containsClass(helpComponent(page), className);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(<Help> {children} </Help>);
+
+      await helpComponent(page).hover();
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toBeVisible();
+      await expect(tooltip).toHaveText(children);
+    });
+  });
+
+  testData.forEach((helpId) => {
+    test(`should check id as ${helpId}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest helpId={helpId} />);
+      await expect(helpComponent(page)).toHaveId(helpId);
+    });
+  });
+
+  ["https://carbon.sage.com", "https://www.google.com"].forEach((hrefLink) => {
+    test(`should check href link as ${hrefLink}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest href={hrefLink} />);
+      await expect(helpComponent(page)).toHaveAttribute("href", hrefLink);
```

Comment:
**suggestion: (non-blocking)** as above, it would be better to add a space to separate the mount from the assertion to follow the same pattern as with the other test cases.